src/objectstyle.rs

```diff
@@ -4,41 +4,44 @@ use std::fmt::Display;
 
 use super::{Attribute, Color, StyledObject};
 
-/// Struct that contains the style properties that can be applied to a displayable object.
+/// An object style.
 #[derive(Debug, Clone, Default)]
 pub struct ObjectStyle {
+    /// The foreground color.
     pub fg_color: Option<Color>,
+    /// The background color.
     pub bg_color: Option<Color>,
+    /// List of text attributes.
```

Comment:
take it or leave it, this type doesn't know anything about text

```suggestion
    /// List of attributes.
```

Reply:
Fixed.